recipes/plotly-resampler/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "plotly-resampler" %}
+{% set version = "0.2.4" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/plotly-resampler-{{ version }}.tar.gz
+  sha256: c31ea7d47dec2675c902440b2caaa966e0e7492df9370f81af40e62588d33f94
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - poetry
+    - python >=3.7
+  run:
+    - dash >=2.0.0
+    - dash-bootstrap-components >=0.13.1
+    - jupyter-dash >=0.4.0
+    - lttbc >=0.2.0
+    - orjson >=3.6.4
+    - pandas
+    - plotly >=5.3.1
+    - python >=3.7
+    - trace-updater >=0.0.6
+
+test:
+  imports:
+    - plotly_resampler
+    - plotly_resampler.downsamplers
+  commands:
+    - pip list | grep -E '{{ name }}\s+{{ version }}'
+  requires:
+    - pip
+    - m2-grep  # [win]
+
+about:
+  home: https://github.com/predict-idlab/plotly-resampler
+  doc_url: https://predict-idlab.github.io/plotly-resampler/
+  summary: Visualizing large time series with plotly
+  license: MIT
```

[This](https://github.com/predict-idlab/plotly-resampler/blob/main/LICENSE) is NOT an MIT license...I'm not sure what it is though.

---

This seems...problematic:
> Usage by non-academic parties is allowed in a strict research
   environment only. The use of the results of the research for commercial
   purposes or inclusion in commercial activities requires the permission of
   Ghent University and IMEC vzw.

---

Oh dang, should have checked... Should I just give up on this?

---

It can go in still, just need accurate metadata. Not sure if it changes whether you're interested in this project.

```suggestion
  license: LicenseRef-plotly-resampler
```

---

Sure, I think it's still interesting and useful in some application